src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -117,7 +117,8 @@ export class HuiThermostatCard extends hassLocalizeLitMixin(LitElement)
           large: this._broadCard!,
           small: !this._broadCard,
         })}">
-        <div id="root">
+        <div id="root" style="min-height: ${(this.clientWidth / 3) * 2 +
```

Comment:
We should not use clientWidth inside the render method. Really bad for performance: https://gist.github.com/paulirish/5d52fb081b3570c81e3a